client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
I think we need a structure either way. One string to define actual theme name (internal/file), other with display friendly name. This would slowly prepare us for a better theme system.